test/end-to-end.js

```diff
@@ -19,6 +19,12 @@ var TestDataBuilder = require('./helpers/test-data-builder');
 var ref = TestDataBuilder.ref;
 var given = require('./helpers/given');
 var should = require('chai').should();
+// this may not work when the scaffolded application uses a different
```

Comment:
```js
// This...
```

to make this consistent with the third line of the comment, which starts with capital letter too